### ISSUE TITLE
What is the "Debug dropdown"?

### ISSUE THREAD

**Issue by alessandro-gentilini** [MicrosoftDocs/cpp-docs#405]
I am trying to follow "Building CMake projects" https://docs.microsoft.com/en-us/cpp/ide/cmake-tools-for-visual-cpp?view=vs-2017#building-cmake-projects but I am lost at "Select the target in the Debug dropdown" because I do not know what is the "Debug dropodown". May you please add a screenshot to help me? Thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 781298c5-3bb4-0bd4-06ca-b09517a76ebc
* Version Independent ID: acb60102-701d-ccb2-d138-2ceb35813292
* Content: [CMake Projects in Visual C++](https://docs.microsoft.com/en-us/cpp/ide/cmake-tools-for-visual-cpp?view=vs-2017)
* Content Source: [docs/ide/cmake-tools-for-visual-cpp.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/ide/cmake-tools-for-visual-cpp.md)
* Product: **visual-cpp**
* GitHub Login: @mikeblome
* Microsoft Alias: **mblome**

**Comment by mikeblome** [comment]
@alessandro-gentilini Sorry about that! Yes I will add a picture. In the meantime, the "dropdown" is the button in the middle of the toolbar with a green arrow that means "start debugging." There is also a small black arrow pointing down. That's the "dropdown" part. Click it to see the different targets that are available.

**Comment by uecasm** [comment]
FYI, when I opened a CMakeLists.txt file just now it only created a single configuration "x64-Debug (default)".

When I selected CMake -> Change CMake Settings -> projectname, it popped up a dialog that allowed me to select other configurations; I chose x64-Debug again and it deleted that previous configuration and created a new one called "x64-Debug" (which now compiled to a slightly different place than before).

Selecting the menu option again does absolutely nothing.  I eventually found that I could right-click the CMakeSettings.json file and select Add Configuration to bring back the dialog and add configurations one at a time (which was mildly annoying since I wanted debug+release+x86+x64, so four in total, and it reminded me to save changes to the settings file for each addition).
